src/components/TabMenu/Item/style.css

```diff
@@ -0,0 +1,20 @@
+.TabItem {
+  color: var(--rvr-gray-lite-1);
+  font-weight: normal;
+  padding: 0;
+  margin: 0 12px;
+  display: block;
+  border-bottom: 2px solid transparent;
+  box-sizing: content-box;
```

Comment:
Why `content-box`? Seems unusual to explicitly have this enabled?

Reply:
So that the bottom border doesn't rest above any parent border styles